src/test/java/bio/terra/workspace/service/datareference/DataReferenceServiceTest.java

```diff
@@ -63,25 +61,41 @@ public void testCreateDataReference() {
     assertThat(ref.name(), equalTo(request.name()));
   }
 
-  @Test
-  public void testCreateInvalidDataReferenceNameFails() {
-    UUID workspaceId = createDefaultWorkspace();
-    DataReferenceRequest request =
-        defaultReferenceRequest(workspaceId).name("!!!!!!!!INVALID NAME!!!!!!!!1").build();
-    assertThrows(
-        InvalidDataReferenceException.class,
-        () -> dataReferenceService.createDataReference(request, USER_REQUEST));
-  }
-
-  @Test
-  public void testCreateDataReferenceNameTooLongFails() {
-    UUID workspaceId = createDefaultWorkspace();
-    DataReferenceRequest request =
-        defaultReferenceRequest(workspaceId).name("1".repeat(100)).build();
-    assertThrows(
-        InvalidDataReferenceException.class,
-        () -> dataReferenceService.createDataReference(request, USER_REQUEST));
-  }
+  // TODO(zloery): reference validation happens at the controller level, so these need to be
+  //    rewritten as integration tests.
```

Comment:
I'm not sure we get a lot of value of keeping the commented out tests or the TODO here. When we fix the TODO, we're not going to change this file. Can we delete the code from this file and either a) file a ticket for adding DR snapshot validation integration tests or b) move this TODO to some integration test file? In the ticket, we can refer to git version that still has these tests.

Reply:
SGTM, created PF-245 to track this and pointed it at the existing code.